src/dbus_api/api/shared.rs

```diff
@@ -130,7 +130,7 @@ pub fn set_key_shared(
     let message: &Message = m.msg;
     let mut iter = message.iter_init();
 
-    let key_desc: &str = get_next_arg(&mut iter, 0)?;
+    let key_desc_str: &str = get_next_arg(&mut iter, 0)?;
```

Comment:
For future reference, if you change this line to `key_desc_string: String`, `get_next_arg` will parse a `String` for you and you can drop the `.to_owned()` below.

Reply:
I see, because `get_next_arg` returns `Result<T, MethodErr>`. @jbaublitz I can change this if you'd like.

Reply:
Unless @mulkieran has objections, I'm happy to keep it the way it is!

Reply:
@lleshchi: Actually, you can get the benefit of that property in two places, so you might as well make the change.